functions/venue.js

```diff
@@ -193,7 +193,8 @@ const createVenueData = (data, context) => {
     owners,
     profile_questions: data.profile_questions,
     placement: { ...data.placement, state: PlacementState.SelfPlaced },
-    showLiveSchedule: data.showLiveSchedule ? data.showLiveSchedule : false,
+    // @debt find a way to share src/settings with backend functions, then use DEFAULT_SHOW_SCHEDULE here
+    showSchedule: data.showSchedule ? data.showSchedule : true,
```

Comment:
is this logic correct?
what if `showSchedule` is `false`? it'll be set to `true` then. Looks like it can will never be `false` here. 🤔 

shouldn't it be
```suggestion
    showSchedule: data.showSchedule ?? true,
```

Reply:
Unfortunately the backend functions are using JavaScript rather than TypeScript at the moment (I have an outdated WIP draft PR to resolve that.. one day...) so we don't have access to using `??` I don't believe.

You're right in saying that the code here seems like a bug though.. Really what it should be is something like:
```
typeof data.showSchedule === 'boolean' ? data.showSchedule : true
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1657/commits/987fd5b68c808862c6e849f92e14d10391b26b14